README.md

```diff
@@ -35,3 +35,12 @@ Make sure the service is up first using `docker-compose up` then run:
 ```sh
 docker-compose exec core python manage.py test
 ```
+
+to run unit tests. Or, to load cancer-static data:
+
+```
+docker-compose exec core bash
+python manage.py acquiredata 
+python manage.py loaddata
+curl http://0.0.0.0:8000/diseases/
```

Comment:
I'd move this out into a separate code block with some text like, "to try it out" or "to see if it's working". It might also be better for the user to do it from their host machine. `curl http://0.0.0.0:8000/diseases` should work on the host, `curl http://localhost:8080/diseases` should let it go through nginx